paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultPaymentSheetLoaderTest.kt

```diff
@@ -49,7 +49,10 @@ internal class DefaultPaymentSheetLoaderTest {
 
     private val customerRepository = FakeCustomerRepository(PAYMENT_METHODS)
     private val lpmRepository = LpmRepository(
-        LpmRepository.LpmRepositoryArguments(ApplicationProvider.getApplicationContext<Application>().resources)
+        arguments = LpmRepository.LpmRepositoryArguments(
+            resources = ApplicationProvider.getApplicationContext<Application>().resources,
+        ),
+        lpmInitialFormData = LpmRepository.LpmInitialFormData(),
```

Comment:
Can't use the default `LpmInitialFormData.Instance` here because other tests might write into that.